### PR TITLE
Add @method to CrudField and CrudColumn for autocomplete

### PR DESCRIPTION
**Pull request by lotarbo**
@tabacitu, hi, i think it may be convenient for developers. Do i need to do this for CrudColumn too?

### CODE REVIEW

**Comment by tabacitu** [comment]
Great idea @lotarbo ! Yes, please do - I think it’d be useful for CrudColumn too. Even Widget.

Out of curiosity, what IDE/TextEditor are you using? How does this help it with autocomplete? Sublime Text 3 here, most of the time.

File: src/app/Library/CrudPanel/CrudField.php [hunk 1]
```diff
@@ -32,7 +42,7 @@ public function __construct($name)
             $this->setAttributeValue('name', $name);
         }
 
-        return $this->save();
+        $this->save();
```
Comment (tabacitu) [inline src/app/Library/CrudPanel/CrudField.php:45]: Did you intentionally remove the return here or was it a mistake? If so why? Thanks!

Comment (lotarbo) [reply src/app/Library/CrudPanel/CrudField.php:45]: `__construct` dont need return and cannot return, its void.

**Comment by lotarbo** [comment]
@tabacitu PhpStrom) 
**Before** https://recordit.co/q42XQmIbFA
**After** https://recordit.co/VrB1qTV6wR

File: src/app/Library/CrudPanel/CrudField.php (hunk 1 above)
Comment (tabacitu) [reply src/app/Library/CrudPanel/CrudField.php:47]: Fair enough! 😄

**Comment by tabacitu** [comment]
Awesome! Thanks a lot @lotarbo , merging now, tagging later today!

**Pull request merged**